mlreco/models/layers/common/momentum.py

```diff
@@ -48,12 +48,12 @@ def __init__(self, num_input, num_output=1, num_hidden=128):
                                                    num_hidden=num_hidden, 
                                                    positive_outputs=False)
     def forward(self, x):
-        if x.shape[0] > 1:
-            self.norm1(x)
+        # if x.shape[0] > 1:
+        #     self.norm1(x)
         x = self.linear1(x)
         x = self.lrelu(x)
-        if x.shape[0] > 1:
-            x = self.norm2(x)
+        # if x.shape[0] > 1:
+        #     x = self.norm2(x)
```

Comment:
Doesn't this break the existing weights?

Reply:
Ah right since we don't use vertex net anyway I'll revert this back

Reply:
Wait I made a separate `VertexNet` to avoid this issue, so I think it's ok, unless we want to keep whatever was trained for vertex MLPs.

Reply:
Oh right I forgot about `VertexNet`. Okay then it should be okay.